### PR TITLE
DHFPROD-3376: mapping UI- If there is no ingested data, there should be an informational message explaining that the query didn't return any source documents

### PR DESCRIPTION
**Pull request by xnikhil08**
Contains the changes for the story DHFPROD-3376.

### CODE REVIEW

File: web/src/main/ui/app/components/flows-new/edit-flow/mapping/ui/mapping-ui.component.html [hunk 1]
```diff
@@ -1,7 +1,9 @@
 <!-- START NEW MAPPING VIEW -->
 <div *ngIf="isVersionCompatibleWithES" class="source-prop-container-map-exp">
-
-  <div>
+  <div *ngIf="targetEntity && !targetEntity.hasDocs">
+    <em>Unable to find documents that match your query.</em>
+  </div>
```
Comment (srinathgit) [inline web/src/main/ui/app/components/flows-new/edit-flow/mapping/ui/mapping-ui.component.html]: In the old mapping  UI, if no docs are found, the info we see is "Before configuring this mapping you must ingest source documents." . Can we continue to have that or is there any reason for the change ?

Comment (bsrikan) [reply web/src/main/ui/app/components/flows-new/edit-flow/mapping/ui/mapping-ui.component.html]: I checked the message suggested in the user story and it matched the implementation here.

File: web/src/main/ui/app/components/flows-new/edit-flow/mapping/ui/mapping-ui.component.html [hunk 2]
```diff
@@ -1,7 +1,9 @@
 <!-- START NEW MAPPING VIEW -->
 <div *ngIf="isVersionCompatibleWithES" class="source-prop-container-map-exp">
-
-  <div>
+  <div *ngIf="targetEntity && !targetEntity.hasDocs">
+    <em>Unable to find documents that match your query.</em>
```
Comment (wooldridge) [inline web/src/main/ui/app/components/flows-new/edit-flow/mapping/ui/mapping-ui.component.html]: @eltesoro-ml has updated the messaging, see the Jira ticket for the new version.

Comment (xnikhil08) [reply web/src/main/ui/app/components/flows-new/edit-flow/mapping/ui/mapping-ui.component.html]: Updated the message as per the Jira tifcket.

**Pull request merged**